### PR TITLE
feat(VIPER-86): design the shopping cart interface

### PR DESCRIPTION
**Pull request by Tatchuong**

### CODE REVIEW

File: src/components/SelectAdress/AddressCard.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  address = { title: '', type: '', address: '', phone: '' },
+  isSelected = false,
+  onSelect = () => {},
+  onEdit = () => {},
+  onDelete = () => {},
+}) => {
+  return (
+    <div
+      style={{
+        border: isSelected ? '2px solid #4CAF50' : '1px solid #ddd',
```
Comment (thienbuidev) [inline src/components/SelectAdress/AddressCard.tsx:11]: using className

File: src/components/SelectAdress/AddressCard.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        borderRadius: '8px',
+        marginBottom: '15px',
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        boxShadow: '0px 4px 6px rgba(0, 0, 0, 0.1)',
+      }}
+    >
+      {/* Address Details */}
+      <div style={{ flex: 1, marginRight: '20px' }}>
```
Comment (thienbuidev) [inline src/components/SelectAdress/AddressCard.tsx:22]: className

File: src/components/SelectAdress/AddressCard.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        <h4 style={{ margin: '0 0 5px', fontSize: '18px', fontWeight: 'bold' }}>
+          {address.title}{' '}
+          <span
+            style={{
+              padding: '4px 8px',
```
Comment (thienbuidev) [inline src/components/SelectAdress/AddressCard.tsx:27]: className

File: src/components/SelectAdress/AddressCard.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+              backgroundColor: 'black',
+              borderRadius: '4px',
+              fontSize: '12px',
+              color: 'white',
+            }}
+          >
+            {address.type}
+          </span>
+        </h4>
+        <p style={{ margin: '5px 0', color: '#666', fontSize: '14px' }}>{address.address}</p>
```
Comment (thienbuidev) [inline src/components/SelectAdress/AddressCard.tsx:37]: className

File: src/components/SelectAdress/AddressCard.tsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        <p style={{ margin: '5px 0', color: '#666', fontSize: '14px' }}>{address.phone}</p>
+      </div>
+
+      {/* Radio Button */}
+      <input
+        type="radio"
+        checked={isSelected}
+        onChange={onSelect}
+        style={{ marginRight: '20px', cursor: 'pointer', width: '18px', height: '18px' }}
```
Comment (thienbuidev) [inline src/components/SelectAdress/AddressCard.tsx]: className

File: src/components/SelectAdress/AddressCard.tsx [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+      />
+
+      {/* Edit/Delete Icons */}
+      <div style={{ display: 'flex', alignItems: 'center' }}>
+        <button
+          onClick={onEdit}
+          style={{
+            background: 'none',
```
Comment (thienbuidev) [inline src/components/SelectAdress/AddressCard.tsx]: className

File: src/components/SelectAdress/AddressCard.tsx [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+            cursor: 'pointer',
+            marginRight: '10px',
+            fontSize: '16px',
+          }}
+        >
+          ✏️
+        </button>
+        <button
+          onClick={onDelete}
+          style={{
```
Comment (thienbuidev) [inline src/components/SelectAdress/AddressCard.tsx]: className

**Review by thienbuidev — CHANGES_REQUESTED** [review]

File: src/components/SelectAdress/AddressCard.tsx [hunk 8]
```diff
@@ -0,0 +1,80 @@
+const AddressCard = ({
```
Comment (bxthien) [inline src/components/SelectAdress/AddressCard.tsx:6]: using interface

**Review by bxthien — CHANGES_REQUESTED** [review]

File: src/components/SelectAdress/AddressCard.tsx [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+        <h4 className="text-lg font-bold mb-1">
+          {address.title}{' '}
+          <span className="px-2 py-1 bg-black text-white rounded text-xs">{address.type}</span>
+        </h4>
+        <p className="text-sm text-gray-600 mb-1">{address.address}</p>
+        <p className="text-sm text-gray-600">{address.phone}</p>
+      </div>
+
+      {/* Radio Button */}
+      <input
```
Comment (bxthien) [inline src/components/SelectAdress/AddressCard.tsx]: using Input Ant design

File: src/components/SelectAdress/AddressCard.tsx [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+        type="radio"
+        checked={isSelected}
+        onChange={onSelect}
+        className="mr-5 cursor-pointer w-4 h-4"
+      />
+
+      {/* Edit/Delete Icons */}
+      <div className="flex items-center">
+        <button
```
Comment (bxthien) [inline src/components/SelectAdress/AddressCard.tsx]: using Button Ant design

File: src/components/payments/Cart.tsx [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+
+  return (
+    <div className="p-5 max-w-5xl mx-auto">
+      <h2 className="text-2xl font-bold mb-4">Cart</h2>
+      <div className="text-sm text-gray-500 mb-5">Home / Cart</div>
+
+      {/* Table Section */}
+      <table className="w-full border-collapse mb-8">
+        <thead>
+          <tr>
```
Comment (bxthien) [inline src/components/payments/Cart.tsx]: using table

**Review by bxthien — CHANGES_REQUESTED** [review]

**Review by thienbuidev — APPROVED** [review]

**Review by thienbuidev — APPROVED** [review]

**Review by thienbuidev — APPROVED** [review]

**Review by thienbuidev — APPROVED** [review]

**Review by bxthien — APPROVED** [review]

**Pull request merged**